modules/lite/src/deckgl.js

```diff
@@ -107,6 +117,17 @@ export default class DeckGL extends Deck {
   }
 
   setProps(props) {
+    const viewState = getViewState(props);
+    if (viewState) {
+      props.viewState = viewState;
+    }
+
+    if ('onViewStateChange' in props && this._updateViewState) {
```

Comment:
When is this._updateViewState not set?

Reply:
In the constructor. `_updateViewState` must be bound to `this`, but we don't have access to the current instance before calling `super(props)`.

Reply:
Oh I see. 

Maybe add a comment stating that this method is sometimes called before the class has been initialized?

It is a rather subtle thing. As a casual reader, I did not understand why this check was there.